src/slider/style.css

```diff
@@ -0,0 +1,62 @@
+tp-slider {
```

Comment:
@AvinashVinod03 Is this committed by mistake?

Reply:
No @junaidbhura, as this is for the draggable functionality example.